index.html

```diff
@@ -54,5 +54,147 @@ <h1 id="greeting">I’m Andrew. Glad to see you!</h1>
         </li>
       </ul>
     </section>
-  </body>
+    <ul id="works">
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio1.png" alt="" />
+        <p class="projectName_holder">Tonic</p>
+
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio2.png" alt="" />
+        <p class="projectName_holder">Multi-Post Stories</p>
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio3.png" alt="" />
+        <p class="projectName_holder">Tonic</p>
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio4.png" alt="" />
+        <p class="projectName_holder">Multi-Post Stories</p>
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+    </ul>
+      <section id="about">
+     <header class="aboutMyself"> 
+     About Myself    
+     </header>
```

Comment:
- [ ] This is the title/heading for this section and therefore should be in a `hx` tag, not a `header`. I'll suggest you update this and all the titles in this section to `h2 or h3`. :wave: :+1: